tests/e2e/expired_client.go

```diff
@@ -124,19 +119,23 @@ func (s *CCVTestSuite) TestConsumerPacketSendExpiredClient() {
 	s.Require().NotEmpty(consumerPackets)
 	s.Require().Equal(2, len(consumerPackets.GetList()), "unexpected number of pending data packets")
 
-	// try to send slash packet for downtime infraction
-	addr := ed25519.GenPrivKey().PubKey().Address()
-	val := abci.Validator{Address: addr}
-	consumerKeeper.QueueSlashPacket(s.consumerCtx(), val, 2, stakingtypes.Downtime)
-	// try to send slash packet for the same downtime infraction
-	consumerKeeper.QueueSlashPacket(s.consumerCtx(), val, 3, stakingtypes.Downtime)
-	// try to send slash packet for the double-sign infraction
-	consumerKeeper.QueueSlashPacket(s.consumerCtx(), val, 3, stakingtypes.DoubleSign)
+	// TODO: reintroduce this section if SlashPackets that are outdated
+	// or do not correspond to any VSCIds awaiting maturation
+	// should not return an errAck and not close the channel
 
-	// check that the packets were added to the list of pending data packets
-	consumerPackets = consumerKeeper.GetPendingPackets(s.consumerCtx())
-	s.Require().NotEmpty(consumerPackets)
-	s.Require().Equal(4, len(consumerPackets.GetList()), "unexpected number of pending data packets")
+	// try to send slash packet for downtime infraction
+	// addr := ed25519.GenPrivKey().PubKey().Address()
+	// val := abci.Validator{Address: addr}
+	// consumerKeeper.QueueSlashPacket(s.consumerCtx(), val, 2, stakingtypes.Downtime)
```

Comment:
Sending these packets would cause an error in `provider.OnRecvSlashPacket` since the slash packet would be considered outdated.

Reply:
I was also running into problems with the tests in this file when first attempting to solve #546. 

Imo we should change the test behavior instead of commenting out

Reply:
Uncomment the tests and use the following reasoning to populate the `valsetUpdateID` of `consumerKeeper.QueueSlashPacket`:
```golang
// a downtime infraction identified at the current height
// has the infraction height set as follows (cf. SDK slashing module)
infractionHeight := s.consumerCtx().BlockHeight() - sdk.ValidatorUpdateDelay - 1
```

```golang
// for a double-sing infraction at consumer height H
// has the infraction height set as follows (cf. SDK evidence module)
infractionHeight := H - sdk.ValidatorUpdateDelay
```

For both downtime and double-signing, send SlashPackets using the following `valsetUpdateID`
```golang
infractionVSCid := consumerKeeper.GetHeightValsetUpdateID(s.consumerCtx(), uint64(infractionHeight))
```